gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Mon Feb 10 14:13:40 PST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

Comment:
what is the difference between the `-bin` and the `-all` distribution?

Reply:
Not sure -- I didn't update this file manually. I'll revert it.